docs/data/base/components/switch/switch.md

```diff
@@ -1,18 +1,17 @@
 ---
 product: base
-title: React Switch component
+title: Unstyled React Switch component and hook
 components: SwitchUnstyled
 githubLabel: 'component: switch'
 waiAria: https://www.w3.org/TR/wai-aria-practices/#switch
 packageName: '@mui/base'
 ---
 
-# Switch
+# Unstyled switch
 
-<p class="description">The Switch component lets you toggle the state of a single setting on or off.</p>
+<p class="description">The `SwitchUnstyled` component provides users with a toggle button for choosing one from a predefined set of options.</p>
```

Comment:
Is our Switch purely for binary on/off toggling, or is it possible to have more than two options?

Reply:
I guess _all_ switches are _only_ for on/off toggling?! Otherwise, it would be something like a Toggle Button (that can accommodate more than 2 options). 

- Material Design definition: _"Switches toggle the state of a single item on or off."_
- Apple's HIG definition: _"A switch is a visual toggle between two mutually exclusive states — on and off."_

Reply:
Thanks for those definitions! I was pulling from the Wikipedia entry on [Cycle buttons](https://en.wikipedia.org/wiki/Cycle_button) (redirected from Switch) which says "two or more." If ours is strictly on/off toggling then I'll revise this to be more specific.